README.md

```diff
@@ -38,6 +38,12 @@ For reference, here is the wiring diagram to hook up the LED and servo to your R
 > For the servo, note that the red (middle) wire is 5v, the brown wire is ground, and the orange wire is data.
 
 # Bring TJBot to Life
+First, make sure you have configured your Raspberry Pi for TJBot.
+If you already have TJBot files:
+    $ cd tjbot/bootstrap && sudo sh bootstrap.sh
+If it is your first time, just run that command to download and install TJBot:
+    $ curl -sL https://raw.githubusercontent.com/ibmtjbot/tjbot/master/bootstrap/bootstrap.sh | sudo -E bash -
```

Comment:
should we `ibm.biz`-ify this url?  most people probably won't care, some people would consider it super dangerous to run a bash script from a URL redirector.  :D